pkg/webhook/server/tls/file_source_test.go

```diff
@@ -0,0 +1,199 @@
+/*
+Copyright 2020 The Jetstack cert-manager contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tls
+
+import (
+	"crypto/rand"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"io/ioutil"
+	"math/big"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	cmapi "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1alpha2"
+	logtesting "github.com/jetstack/cert-manager/pkg/logs/testing"
+	"github.com/jetstack/cert-manager/pkg/util/pki"
+)
+
+func TestFileSource_ReadsFile(t *testing.T) {
+	dir, err := ioutil.TempDir("", "test-temp-dir-")
```

Comment:
nit: can we specify which test this one is for?

Reply:
Fixed in https://github.com/jetstack/cert-manager/pull/2743/commits/0712d008d938e1ea91e40cac520c7a924789ace1